api/HTMLAreaElement.json

```diff
@@ -369,10 +369,10 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "5.1"
+              "version_added": "10.1"
```

Comment:
Sauce Labs has Safari 10, and https://mdn-bcd-collector.appspot.com/tests/api/HTMLAreaElement/origin passes there. That's what WebKit version mapping also points at.